ttnn/cpp/ttnn/operations/normalization/batch_norm/device/kernels/dataflow/reader_batch_norm.cpp

```diff
@@ -38,7 +38,13 @@ void kernel_main() {
     constexpr auto cb_id_eps = tt::CBIndex::c_4;
 
     cb_reserve_back(cb_id_eps, onetile);
-    fill_with_val_bfloat16(cb_id_eps, eps);
+#ifdef FILL_WITH_VALUE_FLOAT
+    float* float_ptr = reinterpret_cast<float*>(const_cast<long unsigned int*>(&eps));
+    FILL_WITH_VALUE_FLOAT(cb_id_eps, *float_ptr);
```

Comment:
Please *never* use `const_cast`. This should just be
```suggestion
    FILL_WITH_VALUE_FLOAT(cb_id_eps, *reinterpret_cast<const float*>(&eps));
```